### PR TITLE
feat: Remove zoom buttons and change position arrow color

### PR DESCRIPTION
**Pull request by KristianSelnas**
The designers have decided to remove the zoom buttons, as most (all?) users are used to zooming the map by pinching. The position arrow button color should be changed to match the filter button.

https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App?node-id=10140%3A47667&t=O1p8CXNMLZQ0f2cB-1

### CODE REVIEW

**Review by gorandalum — COMMENTED** [review]
The code changes looks good, but I think there should be some documentation of the design decision to remove zoom buttons. Either a summary, link to Slack, link to Figma, or something like it.

File: src/components/map/components/filter/MapFilter.tsx [hunk 1]
```diff
@@ -27,8 +29,10 @@ export const MapFilter = ({onFilterChange}: MapFilterProps) => {
       style={style.filterButton}
       type="inline"
       compact={true}
-      interactiveColor={'interactive_2'}
+      interactiveColor="interactive_2"
       accessibilityRole="button"
+      accessibilityLabel={t(MapTexts.controls.filter.a11yLabel)}
+      accessibilityHint={t(MapTexts.controls.filter.a11yHint)}
```
Comment (gorandalum) [inline src/components/map/components/filter/MapFilter.tsx:35]: Are these necessary? Map is disabled when using screen readers.

Comment (KristianSelnas) [reply src/components/map/components/filter/MapFilter.tsx:35]: Very true. It was done for the zoom buttons, so I just followed the same pattern

File: src/translations/components/Map.ts [hunk 2]
```diff
@@ -13,30 +13,6 @@ const MapTexts = {
...
-    filter: {
-      vehicles: {
-        a11yLabel: _('Sparkesykler', 'Electric scooters'),
-        a11yHint: _(
-          'Aktivér visning av elektriske sparkesykler i kartet',
-          'Show or hide electric scooters in the map',
-        ),
-      },
-    },
-    position: {
```
Comment (KristianSelnas) [inline src/translations/components/Map.ts:32]: `controls.position` is not used anywhere

**Review by gorandalum — APPROVED** [review]

**Pull request merged**